src/test/java/uk/gov/companieshouse/confirmationstatementapi/client/OracleQueryClientTest.java

```diff
@@ -308,36 +310,32 @@ void testGetRegisteredEmailAddress() throws ServiceException {
     }
 
     @Test
-    void testGetRegisteredEmailAddressServiceUnavailable() throws ServiceException {
+    void testGetRegisteredEmailAddressServiceUnavailable() {
```

Comment:
may need to be partially reverted if the method in OracleQueryClient is changed...

Reply:
OracleQueryClient has not changed